docs/getting-started/security-ui.asciidoc

```diff
@@ -110,3 +110,50 @@ The Administration page allows you to view and manage hosts that are running End
 
 [role="screenshot"]
 image::images/admin-ui.png[]
+
+[discrete]
+[[timeline-accessibility-features]]
+== Accessibility features
+
+Accessibility features, such as keyboard focus and screen reader support, are built into the Elastic Security UI. These features offer additional ways for you to navigate the UI and interact with the application.
+
+[discrete]
+[[draggable-timeline-elements]]
+=== Interact with draggable elements
+
+Use your keyboard to interact with draggable elements in the Elastic Security UI:
+
+* Select a draggable element to give it keyboard focus.
```

Comment:
The gif below correctly illustrates selection of a draggable via the mouse.

Consider presenting a keyboard-first approach first, and offering a mouse-driving approach as the alternate here:

> * Select a draggable element to give it keyboard focus.

, because some users may be using keyboard-driven selection as the primary means of selection.

The following example leads with keyboard selection of the draggable, before transitioning to the mouse-driven example and the animated gif:

```
Press the `Tab` key to navigate a page until a draggable field is selected.
 
Alternatively, select a draggable element with the mouse to focus it, as shown in the screenshot below:
```